pkg/gofr/datasource/file/ftp/fs.go

```diff
@@ -76,6 +77,9 @@ func (f *fileSystem) UseMetrics(metrics interface{}) {
 func (f *fileSystem) Connect() {
 	var status string
 
+	ftpBuckets := []float64{.05, .075, .1, .125, .15, .2, .3, .5, .75, 1, 2, 3, 4, 5, 7.5, 10}
```

Comment:
Is this bucket made for ftp only? I mean maybe there could be a default bucket values to reuse by defining a variable in a package?

This is more a question than a blocking comment

Reply:
yeah, the thought process behind making separate buckets (although same value) for each datasource is that in future, we can manipulate buckets to achieve higher precision for data.